app/controllers/editorial/nodes_controller.rb

```diff
@@ -27,9 +27,7 @@ def prepare
       configure_defaults!
       authorize! :create_in, @section
 
-      @node_types = Node.descendants.reject {|klass|
-        [RootNode, SectionHome].include? klass
-      }.map do |klass|
+      @node_types = [GeneralContent].map do |klass|
```

Comment:
I think there's only one choice right now. When we grow the number of node types we need to provide the ability to create, this will grow into presumably a more complex solution.